CONTRIBUTING.rst

```diff
@@ -33,6 +33,19 @@ running:
    cd pyvista
    python -m pip install -e .
 
+Quick Start Development with Codespaces
+---------------------------------------
+A dev container is provided to quickly get started. The default container
+comes with the repository code checked out on a branch of your choice
+and all pyvista dependencies including test dependencies pre-installed.
+In addition, it uses the
+`desktop-lite feature <https://github.com/devcontainers/features/tree/main/src/desktop-lite>`_
+to provide live interaction windows.  Follow directions
+`here <https://github.com/devcontainers/features/tree/main/src/desktop-lite#connecting-to-the-desktop>`_
```

Comment:
A UI design expert commented that it is helpful for readers to explicitly indicate the title of the link when add a link ([Jimenez Gacha & Presedo-Floyd - Accessibility best practices for authoring Jupyter notebooks](https://youtu.be/onjRLjveGxk?si=CkG_sZ_sSTIwB36X)). I didn't know that either, so I will try to do it from now on.
```suggestion
`Connecting to the desktop <https://github.com/devcontainers/features/tree/main/src/desktop-lite#connecting-to-the-desktop>`_
```